docs/sources/writing-guide/shortcodes/index.md

```diff
@@ -143,6 +143,54 @@ This shortcode inserts a lists of links to the page's subpages and includes the
 {{</* section withDescriptions="true"*/>}}
 ```
 
+## `docs/reference` shortcode
+
+The `docs/reference` shortcode offers more flexible linking than the Hugo builtin `relref` shortcode.
+When content from one repository is published to more than one documentation set, the `docs/reference` shortcode lets you specify appropriate links for each doc set.
+
+For example, alerting documentation is sourced from the Grafana repository and published to both Grafana documentation and Grafana Cloud documentation.
+Any link from alerting documentation to dashboard documentation, for example, which is also published to both Grafana and Grafana Cloud, needs to be specific as to which destination is appropriate for the product documentation.
+Grafana Cloud alerting documentation should link to Grafana Cloud dashboard documentation, and Grafana alerting documentation should link to Grafana dashboard documentation.
+
+The content of the `docs/reference` shortcode must be within opening and closing tags.
+
+| Parameter             | Description                                                                                                                                                                                                                                                               | Required |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| `label`               | The label for the shortcode. This is the text that you enter in the link instead of an http link or file path.                                                                                                                                                            | yes      |
```

Comment:
I don't think this will makes sense to somebody who hasn't yet seen a reference style link. There's also nothing to help a reader link this to the text in the example. That is, nothing in the example is obviously a label unless you know about reference style links.

Additionally, I think "instead of an HTTP link or file path" might be confusing. I think that without understanding reference style links, I would replace `[text](http://example.com)` incorrectly with `[text](label)` instead of `[text][label]`.